product_docs/docs/edbcloud/beta/using_cluster/index.mdx

```diff
@@ -2,6 +2,6 @@
 title: "Using Your Cluster"
 ---
 
-In this section, account owners and contributors can learn how to connect, edit, scale, and monitor clusters through the EDB Cloud portal. This section also provides information on how EDB Cloud handles backup and restore.
+In this section, account owners and contributors can learn how to access your database, connect, edit, scale, and monitor clusters through the EDB Cloud portal. This section also provides information on how EDB Cloud handles backup and restore.
```

Comment:
I don't think I'd make this change - it confuses "access" and "connect", and "database" and "cluster". It seems fine without it, even with the new section added.

Reply:
undid change in next commit